cinema/views.py

```diff
@@ -0,0 +1,39 @@
+from django.http import HttpRequest
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from .models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies_list(request: HttpRequest) -> Response:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
```

Comment:
your view supports only 2 methods, thus if it isn't GET, then it's automatically POST, so you are free to remove else statement because all if statements have return